rebar.config

```diff
@@ -154,7 +154,7 @@
   {rebar_faster_deps, {git, "https://github.com/arcusfelis/rebar3-faster-deps-plugin.git",
       {ref, "425c75af169f40b9c1c9e471b941175c4530374f"}}},
   {pc, {git, "https://github.com/blt/port_compiler.git", {ref, "c2f3fb1"}}},
-  {provider_asn1, {git, "https://github.com/knusbaum/provider_asn1.git", {ref, "29f7850"}}},
+  {provider_asn1, {git, "https://github.com/arcusfelis/provider_asn1.git", {ref, "29f7850"}}},
```

Comment:
There seems to be hex package for this plubing: https://hex.pm/packages/provider_asn1 did you try it?

Reply:
What's more, version `0.2.1` of the package seems to correspond the commit used by use (if by any reasons we are not able to use the newest version).